### ISSUE TITLE
NaN gradients

### ISSUE THREAD

**Issue by ch3njust1n** [ch3njust1n/gcn#1]
- Occurs after about 788 epochs on backprop
- Plot gradients across training runs
- Plot gradient variance and mean

Debug with weights and biases
